ice/src/main/java/com/altinity/ice/cli/Main.java

```diff
@@ -428,6 +430,36 @@ void deleteNamespace(
     }
   }
 
+  @CommandLine.Command(
+      name = "alter-table",
+      description =
+          "Alter table schema by adding or dropping columns. Supported operations: add, drop. Example: ice alter-table ns1.table1 --operations '[{\"add\": \"age:int:User age\"}, {\"drop\": \"col_name\"}]'")
+  void alterTable(
+      @CommandLine.Parameters(
+              arity = "1",
+              paramLabel = "<name>",
+              description = "Table name (e.g. ns1.table1)")
+          String name,
+      @CommandLine.Option(
+              names = {"--operations"},
+              required = true,
+              description =
+                  "JSON array of operations: [{'add': 'age:int:User age'}, {'drop': 'col_name'}]")
```

Comment:
It would be best to stick to yaml/json for this as column names may contain colons, future operations may not be so easy to express in colon-separated values, etc.

It would also be nice if operation/files names would follow https://iceberg.apache.org/docs/latest/spark-ddl/#alter-table-rename-column naming.